run.go

```diff
@@ -1130,10 +1130,16 @@ func (b *Builder) Run(command []string, options RunOptions) error {
 	case IsolationChroot:
 		err = chroot.RunUsingChroot(spec, path, options.Stdin, options.Stdout, options.Stderr)
 	case IsolationOCIRootless:
+		var moreCreateArgs []string
```

Comment:
Pain in the but nit:
var moreCreateArgs []string {"--no-new-keyring"}
if options.NoPivot {
			moreCreateArgs = append(moreCreateArgs, "--no-pivot")
}
Saves three lines.

Reply:
But adds a syntax error. See what you mean, though